### ISSUE TITLE
Specify dim in softmax function

### ISSUE THREAD

**Issue by tianlu-wang** [jwyang/fpn.pytorch#22]
If you are using pytorch 0.4, pls specify dim when softmax function is called in fpn.py and rpn_fpn.py, otherwise you will probably get mAP=0 for every category.

**Comment by guanbin1994** [comment]
how to do that?

**Comment by AshStuff** [comment]
@guanbin1994 use F.softmax(tensor, 1) in both `fpn.py` and `rpn_fpn.py`

**Comment by XinZhangRadar** [comment]
I met this issue, it has solved follow @AshStuff  advice.
